packages/icons/tests/chromatic/Icons.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import type { ContextValue } from "react-aria-components";
+import type { ResponsiveProp, ColorValue } from "@hopper-ui/styled-system";
+import * as IconLibrary from "@hopper-ui/icons";
+
+import "./icons.stories.css";
+
+interface IconLibraryType {
+    [key: string]: React.ComponentType<ContextValue<Omit<IconLibrary.IconProps, "src16" | "src24" | "src32" >, SVGSVGElement>>;
+}
+
+interface ListProps {
+    icons: string[];
+    size?: ResponsiveProp<"sm" | "md" | "lg">;
+    color?: ResponsiveProp<ColorValue>;
+}
+
+const List = ({ icons, size, color }: ListProps) => {
+    const listItems = icons.map(name => {
+        const Component = (IconLibrary as unknown as IconLibraryType)[name];
+
+        return (
+            <li className="icons-list__item" key={name} >
+                <Component size={size} color={color} />
+            </li>);
+    });
+
+    return <ul className="icons-list">{listItems}</ul>;
+};
+
+export default {
+    component: List,
+    title: "Icons/Icons"
+} as Meta<typeof List>;
+
+type Story = StoryObj<typeof List>;
+
+export const Sizes: Story = {
```

Comment:
Personal preference, but i like the first story to be named Default, when almost no parameters are passed. In this case, yes we tests sizes, but we'll tests all the sizes in every story, so this one is the default, and the next one is testing the styling

Reply:
funny story, I did have default before. Then something wasn't working and I changed the name and just kept it like that.. I'll change it back :)